app/index.html

```diff
@@ -94,9 +94,6 @@
     <script src="bower_components/patternfly/dist/js/patternfly.js"></script>
     <script src="bower_components/angular-patternfly/dist/angular-patternfly.js"></script>
     <script src="bower_components/uri.js/src/URI.js"></script>
```

Comment:
Hmm. Will look now to see if `uri.js` is diff in this version, because changing to the Gruntfile exclude is causing us to miss these components on install.